template.yaml

```diff
@@ -236,6 +236,14 @@ Resources:
       Timeout: 30
       MemorySize: 1536
       Role: !GetAtt NvaNviRole.Arn
+      Events:
```

Comment:
I think there have been som copy/paste issues between UpsertNviCandidateHandler and IndexNviCandidateHandler in this template. The UpsertNviCandidateHandler properties seem wrong

Reply:
IndexNviCandidateHandler should be the one listening til the dynamoDB events